_private/CreateSession.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE hml>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Create Session</title>
+	<link href="bootstrap/css/bootstrap.min.css" rel="stylesheet">
+	<link href="content/style.css" rel="stylesheet">
+</head>
+
+<body>
+	<nav class="navbar navbar-inverse">
+		<div class="navbar-brand">Buzzer</div>
+	</nav>
+	<div class="container">
+		<div class="row">
+			<div class="col-xs-10 col-xs-offset-1 col-sm-9 col-sm-offset-2">
+				<h2>Create New Session</h2>
+				<form class="form">
+					<div class="form-group row">
+						<label class="col-sm-3 col-form-label" for="session_name">Session Name</label>
+						<div class="col-sm-9">
+							<input class="form-control" type="text" id="session_name" name="session_name" />
```

Comment:
Sorry, I was didn't write any css classes yet, but your right, we need to settle on a standard.

I find it easy to read if classes have a hyphen -, id's have an underscore and javascript functions are camelCase  but happy to work with whatever standard.